celery/backends/dynamodb.py

```diff
@@ -153,6 +154,14 @@ def __init__(self, url=None, table_name=None, *args, **kwargs):
                 secret_access_key=aws_secret_access_key
             )
 
+    @staticmethod
+    def _is_valid_ip(ip):
```

Comment:
I wonder if that's the best way to check it.. or if it's worth delaying for 🤔

Anyway, should not be merged before v5.4 release. We're in RC2 at the moment.

Reply:
I guess I could do 
pattern = re.compile(r'^[a-z]{2}-[a-z]+-[0-9]{1}$')
    if pattern.match(input_string):
      do real dynamodb
    else:
      do local dynamodb
if we trust the AWS region names keeps following the current format of xx-yyyy-n
That would also allow to use non ip hosts names.